### PR TITLE
Remove width hack and clear canvas via clearRect

### PR DESCRIPTION
**Pull request by mdamien**
Following #211 and https://github.com/jacomyal/sigma.js/commit/3397bd53bbec57606314810fbd77801c6269ef90

Via [jsperf.com](http://webcache.googleusercontent.com/search?q=cache:bUqXp7bT824J:jsperf.com/canvas-clearrect-vs-width/9+&cd=1&hl=fr&ct=clnk&gl=fr), we can see that clearRect is ~2 times faster.

### CODE REVIEW

File: src/renderers/sigma.renderers.canvas.js [hunk 1]
```diff
@@ -433,9 +433,9 @@
   sigma.renderers.canvas.prototype.clear = function() {
     var k;
 
-    for (k in this.domElements)
-      if (this.domElements[k].tagName === 'CANVAS')
-        this.domElements[k].width = this.domElements[k].width;
+    for (k in this.contexts) {
```
Comment (sheymann) [inline src/renderers/sigma.renderers.canvas.js]: why this change?

**Comment by sheymann** [comment]
Same perfs on Firefox, but x10 faster on Chrome and IE11 :+1:

File: src/renderers/sigma.renderers.canvas.js (hunk 1 above)
Comment (mdamien) [inline src/renderers/sigma.renderers.canvas.js]: because we have to use the context instead of the domElement to use the drawing functions
